builder/main.py

```diff
@@ -73,7 +73,7 @@ def _parseSdccFlags(flags):
 )
 
 env.Append(
-    ASFLAGS=env.get("CCFLAGS", [])[:],
+    ASFLAGS=["-l", "-s"],
```

Comment:
Does this affect existing projects?

Reply:
The existing project supports codes written in C only. The suggested fix aims to add the support of assembly codes as well. and it will not affect the existing project at all jet it will add a new feature to it.
Please check : [https://github.com/platformio/platform-intel_mcs51/issues/38](url)